src/custom/pages/About/index.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react'
-import useMarkdown from '../../hooks/useMarkdown'
-import { Title, Content, AppBodyMod } from './About.styled'
-import markdownContent from './about.md'
 import ReactMarkdown from 'react-markdown'
+import useFetchFile from '../../hooks/useFetchFile'
+import { Title, Content, AppBodyMod } from './About.styled'
+import content from './About.md'
 
 export default function About() {
+  const { file, error } = useFetchFile(content)
+
   return (
     <AppBodyMod>
       <Title>About</Title>
       <Content>
-        <ReactMarkdown>{useMarkdown(markdownContent)}</ReactMarkdown>
+        <ReactMarkdown>{error ? error : file}</ReactMarkdown>
```

Comment:
that's fine for this PR. other PR could have a better error handling. I wouldn't do it, cause not prio at all. But maybe you can add a TODO: Display a custom error

I think we can have at hand a `ErrorContent` component, that shows a cow with a message.

```
 ____________________________________
/ I can feel something bad happened: \
\ Error getting the about page       /
 ------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```

Reply:
Good idea :)